public/js/search.js

```diff
@@ -504,7 +522,18 @@ var Databases = React.createClass({
                                     className="panel panel-default">
                                     <div
                                         className="panel-heading">
-                                        <h4>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
+                                        <h4 style={{display: "inline"}}>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
+                                        <p className="" style={{display: "inline"}}>
```

Comment:
Sorry, I think left aligned with the header will be better:

<img width="448" alt="screen shot 2018-05-30 at 12 08 31" src="https://user-images.githubusercontent.com/90373/40716851-397ef700-6402-11e8-9b6e-61afe3f68202.png">

```
<div
    className="panel-heading">
    <h4 style={{display: "inline"}}>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
    &nbsp;&nbsp;
    <a disabled={true}
        style={{cursor: "pointer"}}
        onClick={
        _.bind(function () {
        this.handleToggle(category)
        }, this)
        }>
        [Toggle all]
    </a>
</div>
```